### ISSUE TITLE
Missing docs on what a chromedriver is and what to specify

### ISSUE THREAD

**Issue by jakkarth** [sirfoga/pygce#12]
## Expected Behavior

Documentation should include what a chromedriver is and what to specify for that option, and should have a default if possible like the url does.

## Actual Behavior

Neither the readme nor the -h output indicates what goes in this field.

## Steps to Reproduce the Problem

  1. Try to run it without specifying a chromedriver
  1. Try to run it with -h
  1. Try to read the readme file

## Specifications

  - pygce version (check `pip3 show pygce`): 2ca3adc96fe5610a27497746912fd61813198e43
  - Platform (e.g `Windows 8`, `Mac OSX Sierra`, `Ubuntu 18.04` ...): Ubutun 16.04
  - Url of connection (e.g `https://connect.garmin.com`, if you don't know it,
  then it's the stock one: `https://connect.garmin.com`): default
  - Python version (check `python --version`): 2.7.12/3.5.2

**Comment by sirfoga** [comment]
See [the Chromedriver download page](http://chromedriver.chromium.org/downloads)

**Comment by jakkarth** [comment]
Thanks, but this doesn't answer the question. I need to know what to type on the commandline when using pygce, or what it is. Presumably downloading it will be a necessary step, but your link doesn't explain what it is or what to type when using pygce.

**Comment by sirfoga** [comment]
**tl;dr**
put the *path of the chromedriver you just downloaded*


For more info:
> what to type on the commandline

Repository front page --> file called README --> [Usage section](https://github.com/sirfoga/pygce#usage)

**Comment by jakkarth** [comment]
As I mentioned in my original ticket, I've read through the readme file, including the usage section. Nowhere in the readme is it explained how to install the chromedriver dependency, or that it needs to be installed at all. Based on the readme, I had assumed it was installed as part of the setup.py step. You've provided a link to the download page here, which I appreciate, but that information would be very useful in the readme.

Additionally, the usage section of the readme just repeats the information available via -h. What I want is an example invocation with real values provided. Such an example with real values would be very helpful for understanding what to type when trying to use your software.

I'm not trying to argue with you. I am trying to get the program to work, and in doing I've found some places where I think the documentation could be improved. I'm trying to improve the readme so that in the future people will be able to use your program more easily.

**Comment by sirfoga** [comment]
See f2c17e0fdc9bd9c2c11140a151536055a8cca49b
